libfreerdp/utils/cliprdr_utils.c

```diff
@@ -199,15 +197,24 @@ UINT cliprdr_serialize_file_list_ex(UINT32 flags, const FILEDESCRIPTORW* file_de
 		}
 
 		Stream_Write_UINT32(s, file->dwFlags);          /* flags (4 bytes) */
-		Stream_Zero(s, 32);                             /* reserved1 (32 bytes) */
+
+		Stream_Write_UINT32(s, file->clsid.Data1);
+		Stream_Write_UINT16(s, file->clsid.Data2);
+		Stream_Write_UINT16(s, file->clsid.Data3);
+		Stream_Write(s, &file->clsid.Data4, sizeof(file->clsid.Data4));
```

Comment:
Maybe we could add an assert here or (further) above that `sizeof(file->clsid.Data4) == 8`?

Reply:
@pnowack ok, can do that.
@hardening well, it is serializing the data structure, of course the data sizes must match